### ISSUE TITLE
Redefine access policy for user's dashboard ?

### ISSUE THREAD

**Issue by drmalex07** [PublicaMundi/ckanext-publicamundi#165]
The problem is that only a dataset's owner (i.e. creator) can control the ingestion workflow for it's resources. This proves to be problematic in the context of an organization when usually >= 2 people need to cooperate and see the same set of datasets.

I suggest that a user can see the following (of course it should be a subset of what he can actually edit):
  * owned datasets
  * all datasets from organizations (s)he administers

Also, in the ui, another search filter may be added in order to limit the matching datasets.

**Comment by kalxas** [comment]
Yes, this is an important issue, affecting usability of the admin dashboard.
I agree on the proposed changes.
